### PR TITLE
Add MEX SRC driver to ALE with test

### PR DESCRIPTION
**Pull request by kberryUSGS**

### CODE REVIEW

File: ale/drivers/mex_drivers.py [hunk 1]
```diff
@@ -584,3 +584,141 @@ def fikid(self):
               Naif ID code used in calculating focal length
             """
             return spice.bods2c(self.instrument_id)
+
+
+
+class MexSrcPds3NaifSpiceDriver(Framer, Pds3Label, NaifSpice, RadialDistortion, Driver):
+    """
+    Driver for a PDS3 Mars Express (Mex) High Resolution Stereo Camera (HRSC) - SRC image.
```
Comment (scsides) [inline ale/drivers/mex_drivers.py:592]: Since HRSC is defined, may want to define SRC too.

File: ale/drivers/mex_drivers.py [hunk 2]
```diff
@@ -584,3 +584,141 @@ def fikid(self):
...
+        : str
+          Spacecraft name
+        """
+        return self.instrument_host_id
+
+
+    @property
+    def focal2pixel_lines(self):
+        """
+        Expects fikid to be defined. This must be the integer Naif id code of
```
Comment (scsides) [inline ale/drivers/mex_drivers.py:657]: Is this true?

File: ale/drivers/mex_drivers.py [hunk 3]
```diff
@@ -584,3 +584,141 @@ def fikid(self):
...
+        : list<double>
+          focal plane to detector lines
+        """
+        return [0.0, 0.0, 111.111111111111]
+
+
+    @property
+    def focal2pixel_samples(self):
+        """
+        Expects fikid to be defined. This must be the integer Naif id code of
```
Comment (scsides) [inline ale/drivers/mex_drivers.py:673]: Same from above

File: ale/drivers/mex_drivers.py [hunk 4]
```diff
@@ -584,3 +584,141 @@ def fikid(self):
...
+        return 512.0
+
+
+    @property
+    def detector_center_sample(self):
+        """
+        Returns the center detector sample.
+
+        This is
+        different from ISIS's center sample because ISIS line scan sensors use
```
Comment (scsides) [inline ale/drivers/mex_drivers.py:705]: Frame camera

File: tests/pytests/data/H0010_0023_SR2/ATNM_MEASURED_040101_050101_V03_0_sliced_-41001.xc [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+'1245926CA7B^B'
+'124591B6082^B'
+'1^1'
+'5^1'
+END_ARRAY 1 43
+TOTAL_ARRAYS 1
+ ~NAIF/SPC BEGIN COMMENTS~
+This CK is for testing with the image: /scratch/mex2020/H0010_0023_SR2.IMG.cub
+
+This CK was generated using the following command: {}
```
Comment (scsides) [inline tests/pytests/data/H0010_0023_SR2/ATNM_MEASURED_040101_050101_V03_0_sliced_-41001.xc:63]: Do we know if there is a way to have this filled in with the actual command?

File: ale/drivers/mex_drivers.py (hunk 2 above)
Comment (kberryUSGS) [reply ale/drivers/mex_drivers.py:657]: Nope, forgot to edit the docs. Thank you!

**Comment by kberryUSGS** [comment]
[SRC ALE driver](https://trello.com/c/3xz3ssoJ/207-src-ale-driver)

**Comment by kberryUSGS** [comment]
Sorry about the delay here. Hadn't noticed the actual test failure. I tried to fix it quickly, failed, and am now reverting the existing HRSC test back to the old format so this can get merged.

File: tests/pytests/data/H0010_0023_SR2/ATNM_MEASURED_040101_050101_V03_0_sliced_-41001.xc (hunk 5 above)
Comment (kberryUSGS) [reply tests/pytests/data/H0010_0023_SR2/ATNM_MEASURED_040101_050101_V03_0_sliced_-41001.xc:63]: Not sure. None of the other ones have it, though.

**Comment by kberryUSGS** [comment]
I think this is good to merge now. @scsides will you re-review?

**Pull request merged**